doc/nrf/app_dev/device_guides/nrf70/nrf7002eb2_dev_guide.rst

```diff
@@ -9,17 +9,17 @@ Developing with nRF7002-EB II
 
 The nRF7002 :term:`Expansion Board (EB)` II (PCA63571), part of the `nRF70 Series Family <nRF70 Series product page_>`_, can be used to provide Wi-Fi® connectivity to compatible development or evaluation boards through the nRF7002 Wi-Fi 6 companion IC.
 
-You can use the nRF7002-EB II to provide Wi-Fi connectivity to the :zephyr:board:`nrf54h20dk` and :zephyr:board:`nrf54l15dk` board targets.
+You can use the nRF7002-EB II to provide Wi-Fi connectivity to the :zephyr:board:`nrf54l15dk` and :zephyr:board:`nrf54lm20dk` board targets.
 
 .. figure:: images/nRF7002eb2.png
    :alt: nRF7002-EB II
 
    nRF7002-EB II
 
-Pin mapping for the nRF54H20 DK and the nRF54L15 DK
-***************************************************
+Pin mapping for the nRF54L15 DK and the nRF54LM20 DK
+****************************************************
 
-For nRF54H20 DK and nRF54L15 DK, refer to the following tables for the pin mapping for these kits.
+For nRF54L15 DK and nRF54LM20 DK, refer to the following tables for the pin mapping for these kits.
 The board is mounted on the **P1** connector of these DKs
```

Comment:
Is it really "mounted"? Or "may be mounted"? Do we give the impression that this is fixed-mounted, if we write it like this? Minor comment, but would like @bama-nordic and @richabp to check the language.